source/js/components/newsletter-signup/organisms/with-submission-logic.jsx

```diff
@@ -239,8 +238,16 @@ function withSubmissionLogic(WrappedComponent) {
       ) {
         message = (
           <>
-            <p>{getText(`confirm your email opt-in`)}</p>
-            <p>{getText(`manage your subscriptions`)}</p>
+            <p>
+              {gettext(
+                "If you haven’t previously confirmed your opt-in to a Mozilla-related email subscription you may have to do so now. <strong>Please check your inbox or spam filter for an email from us to click and confirm your subscription</strong>."
+              )}
+            </p>
+            <p>
+              {gettext(
+                "If you have already confirmed your opt-in to receive Mozilla-related emails, you can now <a href='https://www.mozilla.org/newsletter/recovery/' target='_blank'>manage your subscriptions</a> and update your email preferences."
```

Comment:
Thanks @robdivincenzo , your approach works, let’s just apply it on these two strings as well

Reply:
Thanks @TheoChevalier! Feel free to give it another go